core/src/docs/rfcs/5444_operator_from_uri.md

```diff
@@ -0,0 +1,144 @@
+- Proposal Name: `operator_from_uri`
+- Start Date: 2024-12-23
+- RFC PR: [apache/opendal#5444](https://github.com/apache/opendal/pull/5444)
+- Tracking Issue: [apache/opendal#5445](https://github.com/apache/opendal/issues/5445)
+
+# Summary
+
+This RFC proposes adding URI-based configuration support to OpenDAL, allowing users to create operators directly from URIs. The proposal introduces a new `from_uri` API in both the `Operator` and `Configurator` traits, along with an `OperatorRegistry` to manage operator factories.
```

Comment:
Hi, Could you clarify why the `OperatorRegistry` is needed?

Reply:
Hi, we need some place to store the mapping from `scheme` to `OperatorFactory`.

Reply:
Got it, thanks. Maybe we should explain `Operatorfactory` first ?

Reply:
Thanks for the suggestion; I will update the RFC.